src/client/datascience/baseJupyterSession.ts

```diff
@@ -432,7 +440,12 @@ export abstract class BaseJupyterSession implements IJupyterSession {
             return true;
         }
         // If we're in notebooks and using Remote Jupyter connections, then never shutdown the sessions.
```

Comment:
out of date comment. But this also feels like a dangerous change if we're suddenly shutting down external jupyter sessions that we weren't previously

Reply:
sounds like a pretty explicit call to shutdown the session, so never mind - probably fine to change the behavior